packages/terra-clinical-header/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## Unreleased
 
+* Changed
+  * Replaced `title` prop to `text` prop to allow users to spread title HTML Attribute as customProps.
+  * Added `level` prop to allow users to render different heading levels.
+  * Added `id` prop which is to be paired with `aria-labelledby` for labelling page regions.
+
```

Comment:
Nit for better clarity of the changes:
```suggestion
* Changed
  * Deprecated `title` prop. This will be replaced by the`text` prop in the next major version.
    
* Added
  * Added `level` prop to allow users to render different heading levels.
  * Added `id` prop which is to be paired with `aria-labelledby` for labelling page regions.
  * Added `text` prop to allow users to spread title HTML Attribute as customProps. This will replace the `title` prop in the next major release.
  
```

Reply:
updated [changelog](https://github.com/cerner/terra-clinical/pull/823/commits/2a0747c1873dccb3053cc005b59cdea87205589c#diff-46d09a951255f1060174456bc00fb8b3aac50ec092250ab3191ea81a008943fa)